sceptre/cli/template.py

```diff
@@ -100,3 +100,65 @@ def estimate_cost_command(ctx, path):
             response = response["Url"]
             webbrowser.open(response, new=2)
         write(response + "\n", 'text')
+
+
+@click.command(name="fetch-remote-template", short_help="Prints the remote template.")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def fetch_remote_template_command(ctx, path):
+    """
+    Prints the remote template used for stack in PATH.
+    \f
+
+    :param path: Path to execute the command on.
+    :type path: str
+    """
+    context = SceptreContext(
+        command_path=path,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
+        options=ctx.obj.get("options"),
+        output_format=ctx.obj.get("output_format"),
+        ignore_dependencies=ctx.obj.get("ignore_dependencies")
+    )
+
+    plan = SceptrePlan(context)
+    responses = plan.fetch_remote_template()
+    output = [template for template in responses.values()]
+    write(output, context.output_format)
+
+
+@click.command(name="diff", short_help="Show diffs with running stack.")
+@click.argument("path")
```

Comment:
The convention, I think is to call this "command-path"

Reply:
It's actually path:

```
▶ grep -r "click.argument.*path" sceptre/cli
sceptre/cli/update.py:@click.argument("path")
sceptre/cli/delete.py:@click.argument("path")
sceptre/cli/describe.py:@click.argument("path")
sceptre/cli/describe.py:@click.argument("path")
sceptre/cli/list.py:@click.argument("path")
sceptre/cli/list.py:@click.argument("path")
sceptre/cli/list.py:@click.argument("path")
sceptre/cli/create.py:@click.argument("path")
sceptre/cli/execute.py:@click.argument("path")
sceptre/cli/policy.py:@click.argument("path")
sceptre/cli/launch.py:@click.argument("path")
sceptre/cli/template.py:@click.argument("path")
sceptre/cli/template.py:@click.argument("path")
sceptre/cli/template.py:@click.argument("path")
sceptre/cli/status.py:@click.argument("path")
```

Reply:
And so it is! I'm totally wrong on this. Sorry for the misdirection.